yarn-project/circuits.js/src/merkle/merkle_tree_calculator.ts

```diff
@@ -23,8 +23,7 @@ export class MerkleTreeCalculator {
 
   computeTree(leaves: Buffer[] = []): MerkleTree {
     if (leaves.length === 0) {
-      // TODO(#4425): We should be returning a number of nodes that matches the tree height.
```

Comment:
If you put "Fixes #4425" in the PR description it auto closes the issue once merged